src/jsMain/kotlin/baaahs/client/WebClient.kt

```diff
@@ -55,6 +57,14 @@ class WebClient(
             pinkyState = newState
             facade.notifyChanged()
         }
+
+        pubSub.subscribe(Topics.autoMode) { newAutoMode ->
+            logger.info { "Successful subscribe then publish" }
+            val newMode = newAutoMode == AutoModeState.On
+            val newSettings = uiSettings.copy(autoMode = newMode)
+            updateUiSettings(newSettings, saveToStorage = true)
```

Comment:
I'd just make this a property on the facade rather than putting it in `UiSettings`, cuz it doesn't need to be persisted across browser reloads — the one source of truth should be the pubsub channel.

```kotlin
var autoModeState = AutoModeState.OFF
pubSub.subscribe(Topics.autoMode) { newAutoMode ->
    automodeState = newAutoMode
    facade.notifyChanged()
}
```